### PR TITLE
Change licenses to tldraw

### PR DESCRIPTION
**Pull request by steveruizok**
This PR updates the licenses across tldraw to a bespoke tldraw license.

The idea here is leverage dual licensing for revenue from companies using tldraw. The source code and its distributions are provided under a non-commercial license (tldraw) while we offer to sell / give out an alternative exclusive-use license for companies who wish to use the product for commercial purposes.

- [x] Add new license
- [x] Change licenses in package.jsons
- [x] Update READMEs
- [x] Update docs (separate repo PR)
- [x] Have alternative license in hand (US)
- [ ] Have alternative license in hand (UK)
- [x] Have sales contract in hand (US)
- [ ] Have sales contract in hand (UK)

### Change Type

- [x] `major` — Breaking change

### CODE REVIEW

File: TRADEMARKS.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+This policy encompasses all trademarks and service marks, whether they are registered or not.
+
+## General guidelines
+
+Whenever you use one of our marks, you must always do so in a way that does not mislead anyone about what they are getting and from whom.
+
+Do not use the tldraw marks in any way that could mistakenly imply that tldraw has reviewed, approved or guaranteed your goods or services. You also cannot use our logo on your website in a way that suggests that your website is an official website or that we endorse your website. You can, though, say you like the tldraw software, that you use tldraw, that the analytics are powered by tldraw or that you participate in the tldraw community.
+
+Do use the "tl" prefix in a way that could mistakenly imply that your product is related to tldraw. For example, an analytics product that uses tldraw should not use the name "tlanalytics".
```
Comment (TodePond) [inline TRADEMARKS.md:22]: Looks like a typo?
```suggestion
Do not use the "tl" prefix in a way that could mistakenly imply that your product is related to tldraw. For example, an analytics product that uses tldraw should not use the name "tlanalytics".
```

File: TRADEMARKS.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+Anyone wishing to use any of tldraw’s trademarks in a manner other than the acceptable uses listed above, including but not limited to marketing, promotion or advertising, or on software derivative of tldraw software, must obtain tldraw’s express, written permission in advance.
+
+To request the use of the trademarks in a manner or for a purpose not expressly permitted in these guidelines, including use for any purpose of the logos, please email [hello@tldraw.io](mailto://hello@tldraw.io) to discuss. If you need clarification on whether your use qualifies, please ask.
+
+## To report misuse
+
+If you want to report misuse of a tldraw trademark, please email [hello@tldraw.io](mailto://hello@tldraw.io).
+
+Last updated: Movember 7 2023
```
Comment (TodePond) [inline TRADEMARKS.md:64]:
```suggestion
Last updated: November 7 2023
```

File: apps/examples/LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
Comment (TodePond) [inline apps/examples/LICENSE.md:21]: I think this needs to be updated to AGPL too

File: README.md [hunk 4]
```diff
@@ -97,15 +97,25 @@ Have questions, comments or feedback? [Join our discord](https://discord.gg/rhsy
...
 
-Please see our [contributing guide](https://github.com/tldraw/tldraw/blob/main/CONTRIBUTING.md). Found a bug? Please [submit an issue](https://github.com/tldraw/tldraw/issues/new).
+The GNU Affero General Public License is a free, copyleft license for software and other kinds of works, specifically designed to ensure cooperation with the community in the case of network server software. The AGPL-3.0 license allows users to use and modify the software as long as they keep it open source and provide any modifications or derivative works under the same license.
+
+## Commercial license
+
+If you wish to use this project in closed-source software or otherwise do not want to comply with the AGPL-3.0, you need to purchase a commercial license. Please contact us at [hello@tldraw.com](mailto:hello@tldraw.com) for more information about obtaining a commercial license.
+
+The dual licensing model supports the development of the project by providing an open source option for those who are contributing back to the community, while also supporting commercial usage for entities that are not able to do so.
```
Comment (ds300) [inline README.md:114]: I think this stuff needs to be much clearer to aid adoption. I'd suggest having just one section called 'Licensing' that gets to the point. Something like

```md
## Licensing

tldraw's source code is published under the restrictive AGPL-3.0 license, which means you must obtain a separate license from us if you wish to use tldraw in a commercial project or any project not published under AGPL-3.0.

Obtain a license by contacting us at [hello@tldraw.com](mailto:hello@tldraw.com). Let us know what you're working on and how tldraw will fit in.

We are quite happy to grant free licenses on a case-by-case basis to people using tldraw for charitable projects, research projects, and non-AGPL open source projects.
```

Comment (steveruizok) [reply README.md]: Good feedback, I've tightened it up.

File: apps/examples/LICENSE.md (hunk 3 above)
Comment (steveruizok) [reply apps/examples/LICENSE.md]: We probably have some other projects that we'll need to update as well. I'd actually like our examples to eventually be made available under MIT, though I'd need to ask our lawyer how exactly that would work.

**Comment by kachkaev** [comment]
👋 folks! What is your latest view on the license for v2.0? I have been trying to understand if I can use tldraw in one React app, but it seems that the decision is blocked by the uncertainty. If there is any up-to-date info in the public, it’d be great if you could share the link 🙌

**Comment by steveruizok** [comment]
Update: we're switching to a bespoke license based on the MIT Scratch license. This should be less onerous than GPL while still protecting the core dual-licensing model! @kachkaev I've emailed you, we'll follow up to work out any details there.

**Review by ds300 — APPROVED** [review]
couple of bits of cleanup

**Comment by ds300** [comment]
yeah this is way clearer and better than AGPL, good call

**Comment by Loki-Afro** [comment]
@ds300 
funnily enough (we) as a commercial AGPL project are quite on the opposite spectrum.

**Pull request merged**